docs/developer/graphql.md

```diff
@@ -5,6 +5,18 @@ Besides being a great query language, GraphQL provides great tool support based
 on strongly-typed schemas, type generation for TypeScript, and many other
 advantages.
 
+The Atomist automation API provides you access to the events and data
+from your development platforms using [GraphQL][graphql], a
+widely-used query language and runtime for APIs.
+
+You can use GraphQL with the Atomist automation API for:
+
+1.  Queries that fetch data directly
+2.  Subscriptions to register the types of events you want to receive
+3.  Mutations to change data and make connections
```

Comment:
is this true? We have very limited mutations available AFAIK. I wouldn't imply that there is a general ability to do this (unless @kipz says otherwise)